src/components/CvDataTable/CvDataTable.vue

```diff
@@ -73,20 +73,20 @@
               <Search16 :class="`${carbonPrefix}--search-magnifier-icon`" />
             </div>
             <label
-              :id="`${uid}-search`"
-              :for="`searchbox-${uid}`"
+              :id="`${cvId}-search`"
```

Comment:
This one was already here using "search" as a prefix so I do not want to change to a suffix.